### DISCUSSION TITLE
install how-to (pls more detailled)

### DISCUSSION THREAD

**Discussion by sudo-joe** [blakeblackshear/frigate#4363] [General]
Hello,

feel very uncomfortable with this docker stuff and I am unable to replicate the suggested steps in:
https://docs.frigate.video/installation

Is there really not a little bit more detailled Install-HowTo available?

Thank you very much!

**Comment by NickM-27** [comment]
The docs more or less assume you have docker installed, for that you'll want to look up how to install with your specific device. 

https://docs.docker.com/get-docker/ is a good place to start
